modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/introspection/RuntimeSnapshot.java

```diff
@@ -505,49 +473,22 @@ public void initOperations() {
             return;
         }
         OperationType[] ops = service.getOperations();
-        for (OperationType op : ops) {
+        // make sure operations are ordered, as service currently returns any order
+        List<OperationType> oops = Arrays.asList(ops);
+        Collections.sort(oops, Comparator.comparing(OperationType::getId));
```

Comment:
`List#sort(Comparator)` is preferred to avoid importing `Collections`.

Reply:
There are lots of calls to Collections#sort for lists in this code (including in this PR), do you mean i should change them all?
Note also i followed the same advice than at https://github.com/nuxeo/nuxeo/pull/3882#discussion_r403069172

Reply:
I think the advice was on comparator no?

I would suggest doing it on the new code and replace old code if wanted.

Reply:
Well it suggested that Collections.sort was ok for lists, still.
But I will do the changes.